roles/qe-server-user/tasks/main.yml

```diff
@@ -13,13 +13,13 @@
   with_items:
    - name: 'usmqe-setup'
      repo: "{{ usmqe_setup_repo }}"
-     version: 'master'
+     version: "{{ usmqe_setup_version }}"
```

Comment:
`version` is slightly strange name (I would think more about tags than branches), but that's just my feeling.